library/src/interactor_impl.cxx

```diff
@@ -308,31 +308,44 @@ class interactor_impl::internals
         self->VTKInteractor->GetRenderWindow()->GetRenderers()->GetFirstRenderer();
 
       bool pickSuccessful = false;
-      double pickedPos[3];
+      double picked[3];
       self->CellPicker->Pick(x, y, 0, renderer);
       if (self->CellPicker->GetActors()->GetNumberOfItems() > 0)
       {
-        self->CellPicker->GetPickPosition(pickedPos);
+        self->CellPicker->GetPickPosition(picked);
         pickSuccessful = true;
       }
       else
       {
         self->PointPicker->Pick(x, y, 0, renderer);
         if (self->PointPicker->GetActors()->GetNumberOfItems() > 0)
         {
-          self->PointPicker->GetPickPosition(pickedPos);
+          self->PointPicker->GetPickPosition(picked);
           pickSuccessful = true;
         }
       }
 
       if (pickSuccessful)
       {
+        /*     pos.--------------------.foc
+         *       /|                   /
+         *      / |                  /
+         *     .--.-----------------.picked
+         * pos1     pos2
+         */
```

Comment:
Not sure how you guys feel about ascii art diagrams, coding style doesn't mention it ;)

Reply:
No problem on my side, it looks good :)